kubelet/deb/build.sh

```diff
@@ -5,22 +5,8 @@ PELION_PACKAGE_NAME="kubelet"
 PELION_PACKAGE_DIR=$(cd "`dirname \"$0\"`" && pwd)
 
 declare -A PELION_PACKAGE_COMPONENTS=(
-    ["https://github.com/PelionIoT/edge-kubelet.git"]="v1.0.0")
+    ["https://github.com/armPelionEdge/edge-kubelet.git"]="v1.1.0")
```

Comment:
Domain is wrong, needs to be updated to PelionIoT.

Reply:
I'd prefer to leave this PR for taking the changes from master => dev.
There are 38 places in the dev branch where the domain name needs to be updated, so those can be done in separate PR.